Misc/ACKS

```diff
@@ -1883,3 +1883,4 @@ Robert Leenders
 Tim Hopper
 Dan Lidral-Porter
 Ngalim Siregar
+Henrik Harutyunyan
```

Comment:
The intended name sorting is alphabetically by last name. Please try to adhere to that.